delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/eventlog/GlobalEventLog.scala

```diff
@@ -0,0 +1,35 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.eventlog
+
+import akka.persistence.query.Offset
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Label
+import ch.epfl.bluebrain.nexus.delta.sdk.model.organizations.OrganizationRejection.OrganizationNotFound
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRejection.ProjectNotFound
+import fs2.Stream
+import monix.bio.{IO, Task}
+
+trait GlobalEventLog[T] {
```

Comment:
IndexingLog ?

Reply:
We are going to use that for SSEs too, that's why we added the generic `T`, so `IndexingLog` is not adequate.

Reply:
We are not sure if this can be applied to SSEs or if another approach will be better.